packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -91,11 +115,42 @@ export const Stackbox = () => {
         closeAction={closeTokenPicker}
         initialFocusRef={searchTokenBarRef}
         isOpen={isTokenPickerOpen}
+        onTokenSelect={isPickingTokenFrom ? setTokenFrom : setTokenTo}
```

Comment:
I would prefer this to be function, as it a condition I would better have it above next to other logic.

Something like this:
```js
const selectToken => {isPickingTokenFrom ? setTokenFrom : setTokenTo)
```

Reply:
I think we should come to an agreement regarding variable creation. 
In a previous comment, I've created some constant variables for text reusability purposes (for `SelectTokenButton` labels and checks), that we suggested to remove to achieve (I guess) code simplicity and a more lean codebase.

But now I see we're creating a lot of variables for simple stuff like:
```ts
  const closeConfirmStack = () => setConfirmStackIsOpen(false);
  const closeTokenPicker = () => setTokenPickerIsOpen(false);
  const openConfirmStack = () => setConfirmStackIsOpen(true);
  const selectToken = isPickingTokenFrom ? setTokenFrom : setTokenTo;
  ```
  
  I don't mind either way on this matter TBH, but I'd suggest taking a decision for code normalization purposes

Reply:
Yeah, we should normalize that.

 We can remove the variables and do it directly on the component props (on simple logic like this).